File: AppLovin MAX Demo App - Java/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/applovin/banner/BannerDemoMenuActivity.java
package com.applovin.enterprise.apps.demoapp.ads.applovin.banner;

import android.content.Intent;

import com.applovin.enterprise.apps.demoapp.ads.applovin.DemoMenuActivity;
import com.applovin.enterprise.apps.demoapp.data.main.DemoMenuItem;

public class BannerDemoMenuActivity
        extends DemoMenuActivity
{
    @Override
    protected DemoMenuItem[] getListViewContents()
    {
        return new DemoMenuItem[] {
                new DemoMenuItem( "Programmatic", new Intent( this, BannerProgrammaticActivity.class ) ),
                new DemoMenuItem( "Layout Editor", new Intent( this, BannerLayoutEditorActivity.class ) ),
                new DemoMenuItem( "Zone Integration", new Intent( this, BannerZoneActivity.class ) ),
        };
    }
}

File: AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/applovin/banners/BannerDemoMenuActivity.kt
package com.applovin.enterprise.apps.demoapp.ads.applovin.banners

import android.content.Intent
import com.applovin.enterprise.apps.demoapp.ads.DemoMenuActivity
import com.applovin.enterprise.apps.demoapp.data.main.DemoMenuItem

class BannerDemoMenuActivity : DemoMenuActivity()
{
    override fun getListViewContents(): Array<DemoMenuItem> = arrayOf(
            DemoMenuItem("Programmatic", Intent(this, BannerProgrammaticActivity::class.java)),
            DemoMenuItem("Layout Editor", Intent(this, BannerLayoutEditorActivity::class.java)),
            DemoMenuItem("Zone Integration", Intent(this, BannerZoneActivity::class.java))
    )
}
